modules/store.js

```diff
@@ -0,0 +1,24 @@
+export default class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
```

Comment:
- [OPTIONAL] Great job so far 👍🏻 but I would suggest using array methods that don't mutate the array, e.g. `concat`, `filter`, `slice` instead of `push`, `pop`, `splice` and so on.For more information, visit this link about [How to Write Remarkably Clean JavaScript Code
](https://www.sitepoint.com/immutable-array-methods-javascript/)
eg:

```javascript
  addBook(title, author) {
    id = generateID(); 
    this.books = this.books.concat({ title, author, author });
    return this.books; 
  }
```